Packs/GitLab/ModelingRules/GitlabModelingrules.xif

```diff
@@ -0,0 +1,16 @@
+[MODEL: model = Audit, dataset = gitlab_gitlab_raw]
+|alter        XDM.Audit.original_event_id=id
```

Comment:
no filter?
```suggestion
alter        XDM.Audit.original_event_id=id
```

Reply:
There wasn't one in the mapping rules I got, Do you want me to ask Dror he already wrote the xql ? @yaakovi